requirements.txt

```diff
@@ -0,0 +1 @@
+pexpect
```

Comment:
Does the build require this module or only the lint? Otherwise we should probably keep it in the workflow only.

Reply:
pytest